packages/common.vm/tools/vm.common/vm.common.psm1

```diff
@@ -1229,18 +1229,22 @@ function VM-Set-Service-Manual-Start {
         [ValidateNotNullOrEmpty()]
         [string]$serviceName
     )
-
     try {
         $service = Get-Service -Name $serviceName -ErrorAction SilentlyContinue
-
         if ($service) {
+            if ($service.Status -eq "Running") {
+                Write-Output "INFO" "Stopping service $serviceName..."
+                Stop-Service -Name $service.Name -Force -ErrorAction Stop
+                Write-Output "INFO" "Service $serviceName has been stopped."
+            }
+            
```

Comment:
I think this line (or a close one) has a trailing whitespace that the linter does not like.